README.md

```diff
@@ -57,48 +60,253 @@ This is work in progress as of 14 Nov 2018:
 [![License](https://img.shields.io/npm/l/@airtasker/spot.svg)](https://github.com/airtasker/spot/blob/master/package.json)
 
 <!-- toc -->
-* [Usage](#usage)
-* [Commands](#commands)
-<!-- tocstop -->
+
+- [Usage](#usage)
+- [Commands](#commands)
+  <!-- tocstop -->
+
 # Usage
 
 To get started and set up an API declaration in the current directory, run:
+
 ```
 npx @airtasker/spot init
 ```
 
 You can then run a generator with:
+
 ```
 npx @airtasker/spot generate --api api.ts
 ```
 
+## `@api`
+
+Define an API. This is required and must only be defined once:
+
+```TypeScript
+import { api } from "@airtasker/spot";
+
+@api({
+  name: "My API",
+  description: "My really cool API"
+})
+class MyAPI {}
+```
+
+| Field         | Description                           |
+| ------------- | ------------------------------------- |
+| `name`        | (**required**) Name of the API        |
```

Comment:
Why are these two required? Just thinking in terms of reducing friction for users who don't need to generate documentation. Description isn't required by OpenAPI3 either: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#infoObject.

This is fine for now, but I'd like to address this in the future. It may be more of a philosophical question, but I think Spot should only require the absolute minimum, which is the structure of the API without any human-readable names or descriptions.

Reply:
Description makes sense to be optional, but I think name should be required. Without a name for the service/api, we can't realistically convert to other forms of contracts e.g. OpenAPI and Pact both require a service name/title (unless of course we give it a default name but could potentially be unsafe when working with a contract broker holding contracts for multiple services)

Reply:
Makes sense.